src/components/languagechooser/languagechooser.jsx

```diff
@@ -21,7 +21,7 @@ class LanguageChooser extends React.Component {
         ]);
     }
     handleSetLanguage (name, value) {
-        jar.set('scratchlanguage', value);
+        jar.set('scratchlanguage', value, {domain: `.${window.location.hostname}`});
```

Comment:
Apparently `.localhost` isn't allowed as a cookie domain. So unfortunately I think we need to make a special case on `localhost` for language choice to keep working locally. It didn't work for me, and then I found [this](https://stackoverflow.com/questions/38669040/share-cookies-to-subdomain-on-localhost).